project-api/project-api-test/src/test/java/io/ballerina/projects/test/plugins/LanguageServerExtensionTests.java

```diff
@@ -190,4 +195,72 @@ public void testOneCompilerPluginWithOneCompletionProvider() {
                 CompletionUtil.LINE_BREAK) && edit.range().startOffset() == nodeAtCursor.textRange().startOffset()
                 && edit.range().length() == 0);
     }
+
+    @Test
+    public void testOneCompilerPluginWithOneCompletionProviderNegative1() {
+        String path = RESOURCE_DIRECTORY.resolve("package_plugin_user_with_completions_5").toString();
+        CompileResult result = BCompileUtil.compileAndCacheBala(path);
+        Project project = result.project();
+        PackageCompilation packageCompilation = project.currentPackage().getCompilation();
+        CompletionManager completionManager = packageCompilation.getCompletionManager();
+        Path filePath = Paths.get(path, "main.bal");
+        DocumentId documentId = project.documentId(filePath);
+        Module module = project.currentPackage().module(documentId.moduleId());
+        Document document = module.document(documentId);
```

Comment:
Shall we move this common logic to a separate method?

Reply:
@mindula  Shall we address this later as we need to get this merged asap?

Reply:
> @mindula Shall we address this later as we need to get this merged asap?

+1